python/ray/air/config.py

```diff
@@ -717,6 +717,8 @@ class RunConfig:
             running in command-line, or JupyterNotebookReporter if running in
             a Jupyter notebook.
         verbose: 0, 1, 2, or 3. Verbosity mode.
+            0 = silent, 1 = default, 2 = verbose.
```

Comment:
I think `1` should describe what level it logs, not that it's the default (if possible)

Reply:
also, what is `3`?

Reply:
Removed 3 (leftover). We may iterate what default/verbose mean (and these are also the enum labels). Let's update this docstring when we're clearer about that.